src/DynamoCoreWpf/DynamoCoreWpf.csproj

```diff
@@ -104,18 +109,33 @@
     <None Remove="Views\Menu\TrustedPathView.xaml" />
     <None Remove="Views\GuidedTour\HtmlPages\Resources\ConnectTheNode.gif" />
   </ItemGroup>
-  <ItemGroup>
+    <ItemGroup Condition=" '$(TargetFramework)' == 'net6.0-windows' ">
+        <PackageReference Include="System.Configuration.ConfigurationManager" Version="5.0.0" />
+        <PackageReference Include="AvalonEdit" Version="6.3.0.90" CopyXML="true" />
+        <PackageReference Include="HelixToolkit" Version="2.24.0" />
+        <PackageReference Include="HelixToolkit.Wpf" Version="2.24.0" />
+        <PackageReference Include="HelixToolkit.Wpf.SharpDX" Version="2.24.0" />
+        <Reference Include="PresentationCore" /> 
```

Comment:
I am not sure that we need this reference anymore for net6-windows + wpf.
I think PresentationCore is added as a reference automatically

Reply:
so what is the strategy here ?
I see you did not replace any nuget packages for net6 (some are not net6 compatible)
Do you want to just get this project to build on net6 and net48 without investigating the dependencies ?

Reply:
yes, I sent the PR to validate initial build.